packages/itwinui-css/src/slider/slider.scss

```diff
@@ -158,6 +160,12 @@ $tick-height: $iui-baseline; // 12px
   .iui-slider-min,
   .iui-slider-max {
     margin-top: $iui-xxs;
+    width: $iui-line-height;
```

Comment:
Is it okay to use `$iui-line-height` here (since it is the width property)? Should I create a new variable just for this (line 163) and line 229? Or is it better to use some other hard coded value here?